src/components/AsyncTypeahead/AsyncTypeahead.js

```diff
@@ -137,6 +137,7 @@ export const AsyncTypeahead = ({
         onClick: setShow,
         onKeyDown: (ev) =>
           handlePopoverNavigation({
+            autoopenOnFocus: true,
```

Comment:
Context:
As I'd originally only used the popovers on the async typeahead component, I had this sort of "feature" where if you focused on the search typeahead component it would auto open (as expected for a search typeahead component, but not for the kabob button of course!) -- so this just rewired the usage to explicitly opt in (it defaults to falsy otherwise)